cmd/minishift/cmd/start_preflight.go

```diff
@@ -414,3 +422,31 @@ func isMounted(driver drivers.Driver, mountpoint string) (bool, error) {
 
 	return true, nil
 }
+
+// checkIsoUrl checks the Iso url and returns true if the iso file exists
+func checkIsoUrl() bool {
+	isoUrl := viper.GetString(configCmd.ISOUrl.Name)
+
+	if isoUrl == "b2d" || isoUrl == "centos" || isoUrl == "minikube" {
```

Comment:
We need the list of short names now at multiple places. We should define it in a single place (eg as slice) and use it where appropriate. We already have with the minikube ISO the case where not all usages where updated.

Reply:
Thanks, There's this file in `pkg/minishift/constants/constants.go` which has these constants, i'll  use that.

Reply:
I agree with @hferentschik. We should have a slice which contains the vallid short names .Recently @budhrg also brought it up.

Reply:
Validations.go

Reply:
slice should be go to constants.go

Reply:
@anjannath 

> There's this file in pkg/minishift/constants/constants.go which already has these constants.

Those are independent constant name. We don't have slice.
May be
```
SupportedISOAlias = []string{B2dIsoAlias, CentOsIsoAlias, MinikubeIsoAlias}
```